src/lib/AsyncReactVTK.js

```diff
@@ -0,0 +1,58 @@
+import { lazy } from 'react';
+
+/**
+ * fakeModules: We need to trick React.lazy into thinking we are 
+ * giving the output of an import("my-module.js") Promise.
+ */
+const asyncComponentBuilder = name => async () => {
+    // eslint-disable-next-line
+    const AsyncReactVTK = import(/* webpackChunkName: "ReactVTK" */ './ReactVTK');
+    const ReactVTK = await AsyncReactVTK;
+
+    const reactComponent = ReactVTK.default[name];
+    const fakeModule = { default: reactComponent };
+    return fakeModule;
+}
```

Comment:
Doesn't need to be changed but could be simplified to something like:

```
const asncComponentBuilder = name => import(/* webpackChunkName: "ReactVTK" */ './ReactVTK').then(ReactVTK => ({ default: ReactVTK.default[name] }))
```

Reply:
The code snippet doesn't seem to work:

![image](https://user-images.githubusercontent.com/21180505/114771600-c2651300-9d3a-11eb-8651-0c96eef26970.png)

Reply:
Ok so this should work:

```js
const asyncComponentBuilder = name => () => import(/* webpackChunkName: "ReactVTK" */ './ReactVTK').then(ReactVTK => ({ default: ReactVTK.default[name] }))
```

I believe it achieves the same thing in a different way, but it comes down to whether to use `Promise.prototype.then()` or `async`/`await`.

Reply:
I would prefer to keep this code as is. Thinking about looking at this code in 6 months after forgetting the details of the implementation, I think I'm more likely to be able to read through the current implementation line by line and follow the train of transformations than I am to be able to parse the one-liner.

Reply:
To be honest I disagree as to what is most readable here but as previously stated I'm fine with the current implementation. Functionally both are equivalent.